projects/storefrontapp-e2e-cypress/cypress/helpers/product-configurator-cart.ts

```diff
@@ -23,6 +23,23 @@ export function clickOnEditConfigurationLink(cartItemIndex: number): void {
     });
 }
 
+/**
+ * Clicks on the 'Edit Configuration' link in add-to-cart dialog.
+ *
+ */
+export function clickOnEditConfigurationLinkInAddToCartDialog(): void {
+  cy.get('cx-added-to-cart-dialog').within(() => {
+    cy.get('cx-configurator-cart-entry-info').within(() => {
+      cy.get('cx-configure-cart-entry')
+        .find('a:contains("Edit")')
+        .click()
+        .then(() => {
+          cy.location('pathname').should('contain', '/cartEntry/entityKey/');
```

Comment:
should we check here that the configuration is in edit mode and  not in read-only mode?

Reply:
Agree! I added the check that verifies if the configuration page is rendered.